ui/src/testing/stubs/workspace-service-stub.ts

```diff
@@ -147,10 +148,31 @@ export class WorkspacesServiceStub {
           }
         });
         if (updateIndex === -1) {
-          const msg = `Error sharing. Workspace with id: ${workspaceId} does not exist.`;
+          const msg = `Error updating. Workspace with id: ${workspaceId} does not exist.`;
+          observer.error(new Error(msg));
+          return;
+        }
+        this.workspaces.splice(updateIndex, 1, this.clone(newWorkspace));
+        observer.complete();
+      }, 0);
+    });
+  }
+
+  cloneWorkspace(workspaceNamespace: string,
+                 workspaceId: string,
```

Comment:
Same comment applies here about indenting vs. alignment. Totally your call.

Reply:
Both of these are generally "valid" styles in my view, so the next tie-breaker would be file consistency. Since the rest of the file already follows this indentation, I'm inclined to keep it for now. If we want to apply this +4 (vs paren aligned) across the codebase, that sounds like a reasonable follow-up - assuming there's agreement.

FWIW I tend to agree with you on this particular case.